resources/artifact-ignores.properties

```
@@ -956,3 +956,7 @@ jackson2-api@2.19.0-404.vb_b_0fd2fea_e10        # https://github.com/jenkinsci/b
 
 # SECURITY-3588
 gatling@136.vb_9009b_3d33a_e
+
+# service discontinued
+# https://groups.google.com/g/jenkinsci-dev/c/8VMFBfcmOu4/m/zbnWkyPdAAAJ
+crossbrowsertesting = https://support.smartbear.com/crossbrowsertesting/docs/about.html
```

If it is preferred to have a link to a web page controlled by the Jenkins project, then this could be updated to
```suggestion
crossbrowsertesting = https://github.com/jenkins-infra/update-center2/pull/870
```

I'm OK with either and am open to other suggestions.